doc/source/toolchain.rst

```diff
@@ -111,27 +114,34 @@ attempting to predict adoption timelines for newer standards.
  <= 2018           C90
  2019              C90 for old code, may consider C99 for new
  2020              C99
- 2020              C11
- ?                 C14 some C++14 features can be used, typically limited by MSVC support
```

Comment:
I _think_  that `C11` here actually means `C++11`, as MSVC has not supported C11 (or even the full C99) until very very recently. However, it would make sense to me that `C++11` is considered ready-to-use after dropping python 2.7 support

Reply:
Similarly, `C14` does not exist as a C-Standard. With the dropping of Python 3.6 (#13081), this should now enable C++14 & C++17 (see below), but this is just my interpretation - not trying to make policy here.

Reply:
Yes exactly, C++11 was meant here.